lib/web_ui/test/canvaskit/frame_timings_test.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.6
+import 'package:test/bootstrap/browser.dart';
+import 'package:test/test.dart';
+import 'package:ui/src/engine.dart';
+import 'package:ui/ui.dart' as ui;
+
+import '../frame_timings_common.dart';
+
+void main() {
+  internalBootstrapBrowserTest(() => testMain);
+}
+
+void testMain() {
+  setUpAll(() async {
```

Comment:
This test gives an error for ios-safari. [logs](https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8867598021470141728/+/steps/felt_ios-safari_test/0/stdout)

```
00:15 [32m+363[0m[33m ~157[0m[31m -1[0m: test/canvaskit/frame_timings_test.dart: (setUpAll) [1m[31m[E][0m[0m                                                                                                                             
  ReferenceError: Can't find variable: WebAssembly
  ../src/engine/canvaskit/initialization.dart 37:5 
```

Can we skip the test? Safari desktop tests are passing btw.

Reply:
Oh, right! Done.